lib/kuber_hex/gateways/authorize_net.ex

```diff
@@ -351,7 +578,7 @@ defmodule Gringotts.Gateways.AuthorizeNet do
     element(:customerIP, opts[:customerIP])
   end
 
-  def join_string(list, symbol) do
+  defp join_string(list, symbol) do
```

Comment:
Just 1 use of `join_string`. This is better:
``` elixir
"#{opts[:payment][:card][:year]}-#{opts[:payment][:card][:month]}"
```

Reply:
no I have used it in the add_credit_card() function too, it may be required in future, so kept it.